packages/amplify-environment-parameters/src/__tests__/environment-parameter-manager.test.ts

```diff
@@ -31,6 +31,7 @@ let ensureEnvParamManager: () => Promise<{ instance: IEnvironmentParameterManage
 
 beforeEach(() => {
   jest.clearAllMocks();
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
seems to work even if misused.
I have not found correct replacement without rebuilding this test.